deploy/sre-pruning/README.md

```diff
@@ -0,0 +1,52 @@
+# Pruning Resources on cluster
+
+This folder is required to cleaning up stuff from the cluster that are left over for some reason
+
+# How to build
+## test manually
+firstly, run the commands manually on the cluster to see what commands need to be run automatically
+
+### extract RBAC rules required
+after the script is done look at all of the command and try to extract the rules you need,
+
+For example the command:
+```
+oc delete job
+```
+requires you the `delete` verb to the `jobs` resource
```

Comment:
i think something is off here in regards to the grammar

Reply:
maybe?
```suggestion
this means you need to add the verb  `delete` to the resource named `jobs`
```

Reply:
@RiRa12621 does that look good?

Reply:
yes, nice